src/main/resources/templates/kotlin-lang/kotlinClassGraphqlResponseProjection.ftl

```diff
@@ -0,0 +1,94 @@
+<#if package?has_content>
+package ${package}
+
+</#if>
+import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseField
+import com.kobylynskyi.graphql.codegen.model.graphql.GraphQLResponseProjection
+<#if equalsAndHashCode>
+import java.util.Objects
```

Comment:
Kotlin code is not necessarily for the JVM, so JDK classes will not be available if for instance we generate code for Kotlin/JS or Kotlin/Native.

Reply:
Maybe we can deal with other Kotlin targets in a future PR.
There might be other things to consider to support other Kotlin targets.

Reply:
At present,we should only consider the JVM, otherwise Scala also has Scala.js , Scala native, the implementation is more complex. @kobylynskyi What about you?

Reply:
I think for now it is ok to have a JVM version of Kotlin (also considering the fact that the name of the library is still graphql-**java**-codegen). So I'm 👌 with it.

Reply:
Yes, otherwise I would like to continue to implement Python and Rust, But the underlying library is based on `graphql -Java`,

Reply:
Agreed. At least for the server-side code generation (which was the initial intent of the lib) we are relying on graphql-java and I don't think this will change anytime soon.